flocker/control/_protocol.py

```diff
@@ -109,6 +109,11 @@ class NodeStateCommand(Command):
     Used by a convergence agent to update the control service about the
     status of a particular node.
     """
+    # FLOC-1513
+    #
+    # Add another argument, an optional `ListOf(Unicode)` representing
```

Comment:
It's AMP.  It can't readily transport arbitrary implementations of an interface.  It could use the `wire_encode` and `wire_decode` to transport one (the only one we care about) specific implementation... Or I could just make it `ListOf(Dataset)`.  `Unicode` is simpler on the wire, though, and there are no plans to use anything from the `Dataset` except the `dataset_id` for now...